WalletWasabi.Gui/Controls/WalletExplorer/BuildTransactionDetails.xaml

```diff
@@ -0,0 +1,41 @@
+﻿<UserControl xmlns="https://github.com/avaloniaui" 
```

Comment:
Move this file to Controls/TransactionDetails/Views

Reply:
If you are always using transactioninfo as a viewmodel, should't this have TransactionInfoView instead?